hack/helmfile.yaml

```diff
@@ -135,6 +135,8 @@ releases:
         value: {{ .Values.dynBackendDomain2 }}
       - name: brig.config.optSettings.setFederationDomainConfigs[4].domain
         value: {{ .Values.dynBackendDomain3 }}
+      - name: mlsstats.federationDomain
+        value: {{ .Values.federationDomain1 }}
```

Comment:
We probably don't need to enable this component in integration tests because it will probably just not run. So this config won't be required.

Reply:
I would need the own domain name here to filter (even though that's an optimisation). Where would I get from in prod?